smallrye-reactive-messaging-kafka-test-companion/src/main/java/io/smallrye/reactive/messaging/kafka/companion/TopicsCompanion.java

```diff
@@ -160,6 +160,28 @@ public void clear(String... topics) {
                 .await().atMost(kafkaApiTimeout);
     }
 
+    /**
+     * Same as {@link #clear(String...)} but does not throw an exception if a topic doesn't exist.
+     *
+     * @param topics the topic names to clear
+     */
+    public void clearIfExists(String... topics) {
+        clearIfExists(Set.of(topics));
+    }
+
+    /**
+     * Same as {@link #clear(String...)} but does not throw an exception if a topic doesn't exist.
+     *
+     * @param topics the collection of topic names to clear
+     */
+    public void clearIfExists(Collection<String> topics) {
+        Set<String> existing = list();
+        existing.retainAll(topics);
+        if (!existing.isEmpty()) {
```

Comment:
I'd assume that if a topic exists among the given list, it'll still be cleared.
This reads like clear all or nothing.

Reply:
Right, sorry for messing that. It's fixed